server/src/main/java/org/opensearch/cluster/coordination/UnsafeBootstrapClusterManagerCommand.java

```diff
@@ -81,6 +82,7 @@ public class UnsafeBootstrapClusterManagerCommand extends OpenSearchNodeCommand
     static final Setting<String> UNSAFE_BOOTSTRAP = ClusterService.USER_DEFINED_METADATA.getConcreteSetting(
         "cluster.metadata.unsafe-bootstrap"
     );
+    static final String REMOTE_CLUSTER_STATE_ENABLED_NODE = "Unsafe bootstrap cannot be performed when remote cluster state is enabled.";
```

Comment:
we need to add more details here. 

Unsafe bootstrap cannot be performed when remote cluster state is enabled as cluster state in the remote is considered the source of truth. In case, you still wish do best effort recovery with unsafe-bootstrap, then please disable the <remote cluster setting name>. For more details, please check the OpenSearch documentation.

Reply:
In documentation, we need to clearly capture that only Index Metadata is backed to remote store in 2.10

Reply:
Updated the message. 
Will explicit mention about index metadata persistence in documentation.